hack/ovn-kind-cno.sh

```diff
@@ -202,17 +217,20 @@ done
 # wait until resources are created
 sleep 30
 
-if ! kubectl wait -n openshift-ovn-kubernetes --for=condition=ready pods --all --timeout=300s ; then
-  echo "OVN-k8s pods are not running"
+# Wait for pods to come up and be ready. Note metrics pods do not currently work.
```

Comment:
is there a bz for tracking why metrics pods are not coming up and excluded from the readiness check?

Reply:
well this script isn't really part of the product or anything, I don't know that we need a bz for it. But someone can feel free to fix it in a future patch.